DESCRIPTION

```diff
@@ -30,7 +30,7 @@ Suggests:
     spelling (>= 2.2.0),
     testthat (>= 3.2.0),
     withr (>= 3.0.0)
-Config/Needs/website: rmarkdown, jsonlite, yaml, gtsummary,
+Config/Needs/website: rmarkdown, jsonlite, yaml, gtsummary, tfrmt,
```

Comment:
Unless you put the vignette in `.Rbuildignore` - every package you use in the vignettes should be added to the Suggests. Otherwise R CMD BUILD (without `--no-build-vignettes` flag) or `pkgdown` would complain.

You might want to exclude a vigniette from the BUILD and add this only to the rendered documentation (which is made in a separate process). In such case, you can skip Suggests and add vignette file(s) to `.Rbuildignore` and also add packages to the `Conig/Needs/website` - that's oftentimes the case for all `xyztemplate` (e.g. `nesttemplate`, `tidytemplate`) and other docs-only packages.

Hope that helps!

Reply:
Thanks @pawelru !

In our case we're using an article, not a vignette. These get rendered to the pkgdown site much like vignettes, but are not bundled with the package like vignettes. If we were to add these pkgs to the Suggests field, R CMD Check would complain that we have packages listed that we're not using.

Since @cicdguy has added these packages to the case image, I think we're good for now with now further changes 🤞🏼🤞🏼

Reply:
Yes, you are right. I haven't noticed that this is actually a subdirectory of `vigniettes/`. In such case everything is good as you said. Sorry about the confusion!